### PR TITLE
Feature/1068 transfer without quotes

### PR DESCRIPTION
**Pull request by vijayg10**
* Removed quotes related required fields in post transfer request in openapi yaml file. We need this because we want to remove the dependency on the previous quote / cached quote in the incoming transfer of scheme adapter. This is to support transfer only requests for performance testing where we want to isolate the transfers only without parties and quotes.
* Refactored the circle ci configuration to resolve some build failures due to custom checkout function

### CODE REVIEW

**Comment by vijayg10** [comment]
https://github.com/mojaloop/sdk-scheme-adapter/pull/90

File: src/simulator/api.yaml [hunk 1]
```diff
@@ -320,13 +320,8 @@ components:
       type: object
       required:
         - transferId
-        - quote
```
Comment (bushjames) [inline src/simulator/api.yaml:323]: Is there a test to prove that allowing incoming requests to this endpoint without these previously required properties does not cause errors? I do not see any unit tests to cover this change.

**Review by bushjames — CHANGES_REQUESTED** [review]
Please add some unit tests that prove the behaviour is not broken when requests with and without these previously required fields are handled by the simulator API.

File: src/simulator/api.yaml (hunk 1 above)
Comment (vijayg10) [reply src/simulator/api.yaml:323]: You are right, there are no unit tests to test this change. I will implement as you suggested. Thanks.

**Review by bushjames — APPROVED** [review]

**Pull request merged**